src/Uno.UWP/Storage/CreationCollisionOption.cs

```diff
@@ -6,9 +6,9 @@ namespace Windows.Storage
 {
 	public enum CreationCollisionOption
 	{
-		OpenIfExists,
-		ReplaceExisting,
 		GenerateUniqueName,
+		ReplaceExisting,
```

Comment:
Should these have specified values and link to docs (https://docs.microsoft.com/en-us/uwp/api/windows.storage.creationcollisionoption?view=winrt-22621)